init.lua

```diff
@@ -427,6 +427,43 @@ local function stamina_globaltimer(dtime)
 	end
 end
 
+local function show_eat_particles(player, itemname)
+	-- particle effect when eating
+	local pos = player:getpos()
+	pos.y = pos.y + 1.5 -- mouth level
```

Comment:
get player properties -> `eye_height` -> subtract a magic number.

Reply:
i changed this to `player:get_properties().eye_height * .923`